configure

```diff
@@ -21750,7 +21750,7 @@ then :
           echo "*** you may also be able to get things to work by modifying LD_LIBRARY_PATH"
 else $as_nop
    echo "*** The test program failed to compile or link. See the file config.log for the"
-          echo "*** exact error that occured. This usually means ESD was incorrectly installed"
+          echo "*** exact error that occurred. This usually means ESD was incorrectly installed"
```

Comment:
Drop this part: it actually comes from acinclude/esd.m4, and is not worth fixing.